url/resources/urltestdata.json

```diff
@@ -8173,5 +8173,48 @@
     "input": "http://foo.0x4.",
     "base": "about:blank",
     "failure": true
+  },
+  {
+    "input": "http://foo.09..",
+    "base": "about:blank",
+    "hash": "",
+    "host": "foo.09..",
+    "hostname": "foo.09..",
+    "href":"http://foo.09../",
+    "password": "",
+    "pathname": "/",
+    "port":"",
+    "protocol": "http:",
+    "search": "",
+    "username": ""
+  },
+  {
+    "input": "http://0999999999999999999/",
```

Comment:
I think this should be failure. https://url.spec.whatwg.org/#ipv4-number-parser step 5 will set R to 8, and 9 is not a radix-8 digit.

Reply:
I think this tests this line of the spec:
"If parts’s size is 1, then return false."
So it wouldn't even get to the IPv4 number parser, IIUC.

Reply:
You're right.  I'm having some trouble reading specs today.